app/views/profiles/_edit_private.haml

```diff
@@ -3,7 +3,13 @@
 -#   the COPYRIGHT file.
 
 %hr
-%h3= t('profiles.edit.your_private_profile')
+%h3{:class=>"inline"}
+  = t('profiles.edit.additional')
+= t('profiles.edit.additional_visibility_text')
+= check_box_tag 'profile[public_details]', true, profile.public_details, {'data-size' => 'mini', 'data-on-text' => t('profiles.edit.public'), 'data-off-text' => t('profiles.edit.private')}
+%span{ :rel => 'facebox', :title => t('profiles.edit.additional_hint') }
```

Comment:
I wiped the facebox mention and it still works the same way on mouse-over. So it probably just not required.